pipelines/jobs/configurations/jdk8u.groovy

```diff
@@ -6,7 +6,8 @@ targetConfigurations = [
         "x64Linux"      : [
                 "hotspot",
                 "openj9",
-                "corretto"
+                "corretto",
+                "alibiba"
```

Comment:
should be called dragonwell?

Reply:
I think so, also alibaba is spelled wrong here